nss-day-cohort-29/nutshell-midnight-dream
# Ac styling chat feature

# Description
Friend Delete added to friends feature
Ability to add a user photo to database at register added to Register feature
Add friend in chatroom feature added to Chat feature 

## Type of change
Please delete options that are not relevant.
- [] Bug fix (non-breaking change which fixes an issue)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout ac-styling-chat-feature`
3. Run grunt
4. Verify that:
- When user registers, you are able to add a link to a photo and that it is added to the database upon registration. 
- When user is in chatroom and clicks on the username of another user, a message prompt asking you to add friend appears...if you accept, the user is added to your friend's list (in connections array in database.json) and an alert of "[username] added!". If you choose "cancel",  alert of "bummertown!" is displayed.
- When in your friends list, you should be able to delete a friend (it should be removed from the DOM and the database.json)

# Checklist:
